staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered_test.go

```diff
@@ -33,8 +36,8 @@ var (
 		close(ch)
 		return ch
 	}()
-	infiniteTimeCh <-chan time.Time = make(chan time.Time)
-	closedTimeCh                    = func() <-chan time.Time {
+	infiniteTimeCh <-chan time.Time
+	closedTimeCh   = func() <-chan time.Time {
```

Comment:
closedTimeCh is not used any more, I think.

Reply:
Done. Also inlined `closedStopCh`.